python/hsfs/core/feature_view_engine.py

```diff
@@ -339,7 +339,7 @@ def _set_event_time(self, feature_view_obj, training_dataset_obj):
 
     def _get_start_time(self):
         # minimum start time is 1 second
-        return 1000
```

Comment:
Is this change necessary?

Reply:
If we eventually will support only 10 digit timestamp yes

Reply:
But now we should any timestamp in second right?